src/atom.rs

```diff
@@ -2,6 +2,8 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
+//! Atoms — Functions to manipulate properties on windows
```

Comment:
We probably won't publish the modules, only reexport types from them so this kind of annotation seems better suited to the structs not modules.